lib/mix/tasks/gen_error_code.ex

```diff
@@ -17,6 +25,8 @@ defmodule Mix.Tasks.ForgeAbi.GenErrorCode do
         true -> handle_default_code(code_name, info)
```

Comment:
we can remove the helper function `handle_default_code` below and only handle `false` cases here